cmd/oras/internal/option/packer.go

```diff
@@ -69,6 +71,35 @@ func (opts *Packer) ExportManifest(ctx context.Context, fetcher content.Fetcher,
 	}
 	return os.WriteFile(opts.ManifestExportPath, manifestBytes, 0666)
 }
+func (opts *Packer) Parse() error {
+	currentDir, err := os.Getwd()
+	var failedPaths []string
+	if err != nil {
+		return err
+	}
+	if !opts.PathValidationDisabled && len(opts.FileRefs) != 0 {
+		for _, path := range opts.FileRefs {
+			//Remove the type if specified in the path <file>[:<type>] format
+			lastIndex := strings.LastIndex(path, ":")
+			if lastIndex != -1 {
+				path = path[:lastIndex]
+			}
+			absPath, err := filepath.Abs(path)
+			dirPath := filepath.Dir(absPath)
+			if err != nil {
+				return err
+			}
+			if dirPath != currentDir {
+				failedPaths = append(failedPaths, absPath)
+			}
+		}
+		if len(failedPaths) > 0 {
+			errorMsg := fmt.Sprintf("%v: %v currentDir :%v", errPathValidation, strings.Join(failedPaths, ", "), currentDir)
```

Comment:
If we don't user %w to wrap an `errPathValidation`, we don't need to declare that err type. We can merge the content of `errPathValidation` into this Sprintf.